cmd/broker/ingress/main.go

```diff
@@ -64,6 +71,8 @@ func main() {
 
 	c := getRequiredEnv("CHANNEL")
 
+	brokerName, _ = os.LookupEnv("BROKER")
```

Comment:
Why make this optional? 

Especially because we will only notice it is missing when we start looking at metrics that we hoped had it.

Reply:
For convenience in debugging when we don't care about metrics. But I'm fine with making it non-optional, since it does makes the normal case safer.

Reply:
This is a main(), which we don't traditionally unit test...

Reply:
This is required now.